docs/building_blocks_for_flexible_content.md

```diff
@@ -0,0 +1,571 @@
+
+### Simple blocks
+
+Simple blocks general render one component or "thing". They can either be rendered directly or as the children of compound blocks.
+
+- [Action link](#action-link)
+- [Big number](#big-number)
+- [Document list](#document-list)
+- [Govspeak](#govspeak)
+- [Heading](#heading)
+- [Image](#image)
+- [Main Navigation](#main-navigation)
+- [Quote](#quote)
+- [Share links](#share-links)
+- [Side Navigation](#side-navigation)
+- [Statistics](#statistics)
+
+#### Action link
+
+A wrapper around the [action link component](https://components.publishing.service.gov.uk/component-guide/action_link).
+
+##### Example
+
+```yaml
+- type: action_link
+  text: "Learn more about our goals"
+  href: "/landing-page/goals"
+```
+
+#### Big number
+
+A wrapper around the [Big number component](https://components.publishing.service.gov.uk/component-guide/big_number)
+
+##### Example
+
+```yaml
+- type: big_number
+  number: £75m
+  label: amount of money that looks big
+```
+
+#### Document list
+
+A wrapper around the [Document list component](https://components.publishing.service.gov.uk/component-guide/document_list)
+
+##### Example
+
+```yaml
+- type: document_list
+  items:
+  - text: An example link
+    path: https://www.gov.uk
+    document_type: Press release
+    public_updated_at: "2016-06-27 10:29:44 +0000"
+  - text: Another example link
+    path: https://www.gov.uk
+    document_type: News article
+    public_updated_at: "2021-01-16 11:34:12 +0000"
+  - text: A third example link
+    path: https://www.gov.uk
+    document_type: Consultation
+    public_updated_at: "2024-02-01 09:00:11 +0000"
+```
+
+#### Govspeak
+
+A wrapper around the [Govspeak content component](https://components.publishing.service.gov.uk/component-guide/govspeak). Note that the content item data for the component is not Govspeak, but HTML that has been converted from Govspeak. If you're editing the block data in Whitehall, though, you can use Govspeak directly by taking advantage of the publishing-api automatic conversion (see second example)
+
+##### Examples
+
+```yaml
+- type: govspeak
+  content: |
+    <h3>Lorem heading</h3>
+    <p>Korem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis
+    molestie, dictum esta, mattis tellus.</p>
+```
+
+```yaml
+- type: govspeak
+  content:
+    content-type: text/govspeak
+    content: |
+      ### Lorem heading
+
+      Korem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis
+      molestie, dictum esta, mattis tellus.
+```
+
+#### Heading
+
+A wrapper around the [Heading component](https://components.publishing.service.gov.uk/component-guide/heading). Note, though, that the block uses `content` where the component uses `text` for the title text key. This is so that headings can be indexed (only values inside a `content` key will be indexed when being published from Whitehall - see [Indexing block content in search](#indexing-block-content-in-search))
```

Comment:
This contextless reference to indexing threw me. The link to indexing helped, but I could have made more sense of this if you specified search here - I was thinking we were numbering headings for some reason as per an array index. Suggestion:
"This is so that headings can appear in search (only values..." etc